docs/guides/envvars.md

```diff
@@ -54,6 +54,27 @@ Note that `dotenv` is only for development, you should not use it in production,
 
 <docs-error>Do not commit your <code>.env</code> file to git, the point is that it contains secrets!</docs-error>
 
+If you're using the `@remix-run/cloudflare-pages` adapter, env variables work a little differently. When running locally, you'll need to manually pass your `.env` file to `wrangler` by doing this:
+
+```sh
+// package.json
+
+{
+	"scripts": {
+		"dev:wrangler": "cross-env NODE_ENV=development wrangler pages dev ./public --binding $(cat .env)"
```

Comment:
aha, I didn't even think to cat the env file, I've been binding these values individually (like a fool)

Reply:
I gotta give a h/t to @theeomm on that one! https://github.com/remix-run/remix/pull/3186#issue-1235573117